src/webui/abstractwebapplication.cpp

```diff
@@ -87,6 +89,15 @@ struct WebSession
 
 // AbstractWebApplication
 
+const QStringMap AbstractWebApplication::CONTENT_TYPE_BY_EXT = {
+    { "htm", Http::CONTENT_TYPE_HTML },
+    { "html", Http::CONTENT_TYPE_HTML },
+    { "css", Http::CONTENT_TYPE_CSS },
+    { "gif", Http::CONTENT_TYPE_GIF },
+    { "png", Http::CONTENT_TYPE_PNG },
+    { "js", Http::CONTENT_TYPE_JS }
```

Comment:
This lookup table is not enough for alternative UI use.
Please add the following types:
* jpg & jpeg: `image/jpeg`
* bmp: `image/bmp`
* txt: `text/plain`
* not found in table: `application/octet-stream`

Reply:
Well. Done.